README.md

```diff
@@ -31,8 +31,13 @@ TODO
 All examples assume a client and/or channel has been created as follows:
 
 ```go
-client := ably.NewRestClient(ably.ClientOptions{ApiKey: "xxxx"})
-channel := client.Channel('test')
+options, err := ably.NewClientOptions("xxx:xxx")
```

Comment:
I think the shorthand initialisation with just a key string is causing too many problems and if overloading is not possible, I would simply go for the following instead:

```
ably.NewRestClient(ably.ClientOptions{ Key: "xxxx" })
```

It reduces complexity significantly.